css/properties/clip-path.json

```diff
@@ -314,7 +314,22 @@
                   "version_added": "71"
                 },
                 {
-                  "version_added": "63",
+                  "version_added": "64",
```

Comment:
@ddbeck FYI
- changes version added to 64, based on tracking flag in bug https://bugzilla.mozilla.org/show_bug.cgi?id=1487838
- Mirrored the release version for FF for Android.